### DISCUSSION TITLE
Please share a doc how woonuxt, woonuxt-settings and woonuxt-setting-module work tobether when you have time?

### DISCUSSION THREAD

**Discussion by mobilelifeful** [scottyzen/woonuxt#185] [Q&A]
Having FQ&A help a bit, but for people not sure where to get started correctly not much. Thank you in advance.

**Comment by scottyzen** [comment]
Hi @mobilelifeful, 

Sure. I'll try to get some documents up on the specifics as soon as I can but in a nutshell, the module is there to load the below data into WooNuxt.

```js
const query = `
query getWooNuxtSettings {
  woonuxtSettings {
    primary_color
    logo
    publicIntrospectionEnabled
    frontEndUrl
    maxPrice
    productsPerPage
    global_attributes {
      slug
      showCount
      openByDefault
      label
      hideEmpty
    }
    stripeSettings {
      enabled
      testmode
      test_publishable_key
      publishable_key
    }
    wooNuxtSEO {
      provider
      url
      handle
    }
  }
  pluginWPGraphQL: plugin(id: "${WPGraphQLPlugin.id}")    { name version }
  pluginWooGraphQL: plugin(id: "${WooGraphQLPlugin.id}")  { name version }
  pluginWPCORS: plugin(id: "${WPGraphQLCors.id}")         { name version }
  generalSettings { title }
}
`;
```

**Reply by mobilelifeful** [reply]
Thank you for such quick response. You will definitely will share a good doc when you have the time. ( i believe) I use local.app for wordpress locally. but did't carry me far, not sure if I get the env. setup correctly or something else. will try again when a doc is available.

So far, using the local wordpress with ssl enable and set GQL_HOST="http://newnew.local" int the .env file.
sure both   woonuxt-settings and woonuxt-setting-module is installed and 

<img width="1272" alt="image" src="https://github.com/scottyzen/woonuxt/assets/91811647/e43232cd-1809-48d5-84eb-a6798ea4f4bd">

then in the woonuxt dir, I run npm install( trying to build the front-end)  Thank you again.

added:

 Error fetching woonuxt settings. Make sure you have the latest version woonuxt-settings plugin installed and activated WordPress. You can download it from https://github.com/scottyzen/woonuxt-settings

 ERROR                                                                                                                                                                             2:53:42 PM
        Failed to load schema from http://newnew.local:
        
        
<img width="474" alt="image" src="https://github.com/scottyzen/woonuxt/assets/91811647/d718ba34-341d-4ff2-a9a3-02ae2b847b08">

**Reply by alexookah** [reply]
I think you should use for GQL_HOST: `http://newnew.local/graphql`.
Make sure you add the path as from the WP_GraphQL Settings:

![Screenshot 2024-07-21 at 02 25 14](https://github.com/user-attachments/assets/8e27eed0-7e60-4d23-9646-326021321366)
